ios/Classes/BetterPlayer.m

```diff
@@ -673,33 +683,16 @@ - (void)usePlayerLayer: (CGRect) frame
     }
 }
 
-- (void)preparePictureInPicture: (CGRect) frame
-{
-    if(_player)
-    {
-        // Create new controller passing reference to the AVPlayerLayer
-        self._playerLayer = [AVPlayerLayer playerLayerWithPlayer:_player];
-        UIViewController* vc = [[[UIApplication sharedApplication] keyWindow] rootViewController];
-        self._playerLayer.frame = frame;
-        self._playerLayer.needsDisplayOnBoundsChange = YES;
-        [vc.view.layer addSublayer:self._playerLayer];
-        vc.view.layer.needsDisplayOnBoundsChange = YES;
-        if (@available(iOS 9.0, *)) {
-            _pipController = NULL;
-        }
-        [self setupPipController];
-    }
-}
-
 - (void)willStartPictureInPicture: (bool) willStart
 {
     self._willStartPictureInPicture = willStart;
+    _pipController = nil;
+    
     if (willStart) {
-        if(!_pipController) {
-            [self preparePictureInPicture:CGRectZero];
-        }
-    } else {
-        _pipController = nil;
+        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(0.2 * NSEC_PER_SEC)),
```

Comment:
Sorry.
What is the mean of 0.2 seconds?
(I'm afraid of causing bugs on faster or slower devices.)

Reply:
@noboru-i 
> What is the mean of 0.2 seconds?

This is a workaround for case change video (change data source), I did try other ways to resolve this but it doesn't work.
I tested it on iPhone 13 mini and it works well.

I guess when changing data source we need to:
- de init old `_pipController`
- wait 0.2s for `_pipController` dispose success (release old video layer, any configure...). It's almost likes this case (setup pip and wait 0.2s to start)
https://github.com/dwango-nfc/betterplayer/blob/be4aa754acddcb0223849e6e611bd9687103d028/ios/Classes/BetterPlayer.m#L675-L679
- create new `_pipController`

Reply:
@mvn-thanhluu-hn 
Thanks. I understand.

Please add more comments.
- "0.2 seconds" is a magic number. But it is the same as the library's code. https://github.com/jhomlala/betterplayer/blob/f6a77cf6fbb515f01aa9fb459b2ee739de3e724c/ios/Classes/BetterPlayer.m#L647
- It is waiting to release the previous `_pipController`.